server/service/software_installers.go

```diff
@@ -6,14 +6,14 @@ import (
 	"encoding/json"
 	"errors"
 	"fmt"
+	"github.com/docker/go-units"
```

Comment:
Please set your IDE to sort/separate standard library packages from external packages.

Reply:
Will do, thanks

Reply:
(yep, I'm using GoLand)
<img width="290" alt="image" src="https://github.com/user-attachments/assets/0721fe71-357e-4580-b6f8-2657069815cc">

Hopefully those settings are correct. Let me know if that, along with e7c96be8540e633be627d2f01e5b8308b416ec77, looks amiss (can reopen if so).